_posts/2025-01-08-joelucas-opensource-experience.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title:  "Data Umbrella Interview: Joe Lucas"
+description: Joe Lucas shares his experience collaborating in open source.
+date:   2024-01-08 15:01:35 +0300
+image:  '/images/blogs/Joe-Lucas-blog.png'
+tags:   [opensource]
+---
+
+Authors:  [Reshama Shaikh](https://reshamas.github.io), [Beryl Kanali](https://github.com/BerylKanali) & [Joe Lucas](https://github.com/JosephTLucas)
+
+<p float="right">
+   <a href=""> </a>
+  <img src="../images/blogs/sprint_joelucas.png" width="85%" height="85%" style="padding:1px;border:thick solid black;" align="top" />
+</p>
+
+1. __Tell us about yourself.__
+
+   I’ve moved all over the United States, but I’m originally from Pennsylvania and now live in Texas with my wife and two daughters. I just bought a pair of cowboy boots, so I guess you can say I’m getting used to the change. I’m currently on the AI Red Team at NVIDIA, which perfectly blends my two professional interests: math and cybersecurity. “Red Teaming” comes from the concept of “red versus blue” where the Blue team consists of developers and security folks, while the Red team thinks and acts like the adversary. By red teaming, we test defensive tools and processes to help provide recommendations. In the end, we’re all contributing towards a more secure product and developer ecosystem. So an AI Red Team performs security reviews, penetration tests, and adversary emulation operations to probe and test the security of MLOps and deployed AI systems. I originally got interested in the machine learning security field when I learned about adversarial images (images that look correct to humans but are intentionally misclassified by classifiers), but the ride has gotten wilder since the most recent AI wave.
+
+2. __How did you learn of the scikit-learn June 2020 (first online) global sprint, organized by Data Umbrella?__
```

Comment:
"scikit-learn June 2020" can you link to this page?
https://sites.google.com/view/nyc-2020-scikit-sprint/